joomla/joomla-cms
# [4.3] Update deleted files and folders lists in script.php for upcoming 4.3.0-beta4

Pull Request for Issue # .

### Summary of Changes

This pull request (PR) updates the list of files to be deleted on update in file "administrator/components/com_admin/script.php" to recent changes in the 4.3-dev branch in preparation for the upcoming 4.3.0-beta4 release.

Currently there is only one file '/layouts/joomla/content/categories_default_items.php' added due to PR #39933 .

In addition it contains the changes from PR #39998 for the 4.2-dev branch. This will lead to a merge conflict when doing an upmerge from 4.2-dev to 4.3-dev after both PR's have been merged into these branches, which can be easily resolved by accepting the 4.3-dev file, or it will lead to a conflict in this PR when the other one is merged before this one.

Finally, it deletes file '/components/com_contact/layouts/field/render.php' in the 4.3-dev branch. This file has been deleted in the 4.2-dev branch with PR #37716 , but this was not merged up correctly into the 4.3-dev branch with the [upmerge last week](https://github.com/joomla/joomla-cms/commit/b9453908eb8ab25f98078e6fd9615674b1095bba) , im opposite to the other file deleted by the same PR which has been correctly deleted in the 4.3-dev branch with that upmerge.

### Testing Instructions

Code review.

Or if you want to make a real test, update a 4.3.0-beta3 to the last 4.3 nightly build to get the actual result, and update a 4.3.0-beta3 or any older 4.x version to the update package built by Drone for this PR to get the expected result.

In addition, follow the testing instructions of PR #37716 .

### Actual result BEFORE applying this Pull Request

The files and folders mentioned above and in PR #39998 are still present after updating from a 4.3.0-beta3 or any older 4.x version.

### Expected result AFTER applying this Pull Request

The files and folders mentioned above and in PR #39998 have been deleted after updating from a 4.3.0-beta3 or any older 4.x version.

PR #37716 still works.

### Link to documentations
Please select:
